sdks/python/apache_beam/io/external/xlang_bigqueryio_it_test.py

```diff
@@ -259,11 +259,11 @@ def test_write_with_beam_rows_cdc(self):
 
     rows_with_cdc = [
         beam.Row(
-            cdc_info=beam.Row(
+            row_mutation_info=beam.Row(
```

Comment:
Let's add an identical test for python dicts (I remember you had one previously with the callable function, but we can leave that part out)